modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/handle/migrate/MigrateDeployProcess.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.workflow.handle.migrate;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.activiti.engine.RuntimeService;
+import org.activiti.engine.delegate.DelegateExecution;
+import org.activiti.engine.delegate.JavaDelegate;
+import org.eclipse.xpanse.modules.deployment.DeployService;
+import org.eclipse.xpanse.modules.models.service.deploy.CreateRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Migration process deployment service processing class.
+ */
+@Slf4j
+@Component
+public class MigrateDeployProcess implements Serializable, JavaDelegate {
+
+    private static RuntimeService runtimeService;
+    private static DeployService deployService;
+
+    @Autowired
+    public void setRuntimeService(RuntimeService runtimeService) {
```

Comment:
instead of setters, please add a constructor.

Reply:
This is very magical. If you use constructor injection, you will not be able to instantiate MigrateDeployProcess, but setter injection is no problem.
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/563f866c-ad30-4b14-aa3c-98434680a675)